packages/@expo/cli/src/run/index.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+import chalk from 'chalk';
+
+import { Command } from '../../bin/cli';
+import { Log } from '../log';
+import { assertWithOptionsArgs } from '../utils/args';
+import { CommandError } from '../utils/errors';
+import { selectAsync } from '../utils/prompts';
+
+export const expoRun: Command = async (argv) => {
+  const args = assertWithOptionsArgs(
+    {
+      // Types
+      '--help': Boolean,
+      // Aliases
+      '-h': '--help',
+      // All other flags are handled by `expoRunAndroid` or `expoRunIos`
+    },
+    {
+      argv,
+      permissive: true,
+    }
+  );
+
+  if (args['--help']) {
+    Log.exit(
+      chalk`
+  {bold Description}
+    Run the native app locally
+
+  {bold Usage}
+    $ npx expo run:android <dir>
+    $ npx expo run:ios <dir>
+
+  {bold Options}
+    --no-build-cache                 Clear the native build cache
+    --no-install                     Skip installing dependencies
+    --no-bundler                     Skip starting the bundler
+    --configuration <configuration>  {underline iOS} Xcode configuration to use. Debug or Release. {dim Default: Debug}
+    --variant <name>                 {underline Android} build variant to use. {dim Default: debug}
+    -d, --device [device]            Device name to run the app on
+    -p, --port <port>                Port to start the dev server on. {dim Default: 8081}
+    -h, --help                       Output usage information
+`,
+      0
+    );
+  }
+
+  const platform = await selectAsync('Select the platform to run', [
+    { title: 'Android', value: 'android' },
+    { title: 'iOS', value: 'ios' },
+  ]);
+
```

Comment:
After this step, it should print the `expo run:<platform>` command with the same flags. So the user knows what command to run next time. cc @EvanBacon